tekton/ci-unit-test.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: unit-tests
+spec:
+  inputs:
+    params:
+    - name: package
+      description: package (and its children) under test
+      default: github.com/tektoncd/pipeline
+    resources:
+    - name: source
+      type: git
+      targetPath: src/${inputs.params.package}
+  #outputs: # coverage file
+  steps:
+  - name: unit-test
+    image: golang:1.12
+    workingdir: /workspace/src/${inputs.params.package}
+    command:
+    - go 
+    args: ["test", "-v", "-cover", "./..."]
```

Comment:
do we want that race condition flag too?

Reply:
and/or a flexible set of flags, maybe with a default?

(need that array type! :D https://github.com/tektoncd/pipeline/issues/207)

Reply:
good point for `race` !
yeah we really need #207 :sweat_smile: as of now, we need a wrapper or some bashism to support flexible set of flags (which reduce a bit readability)

Reply:
Updated (in a separate commit)